agents/ibmz/fence_ibmz.py

```diff
@@ -488,6 +494,14 @@ def set_opts():
         "shortdesc" : "How long to wait for server data",
         "order" : 2
     }
+    all_opt["load_on_activate"] = {
+        "getopt" : "",
+        "longopt" : "load-on-activate",
+        "help" : "--load-on-activate             Rely on the HMC to perform "
+                 "a load operation on activation",
+        "required" : "0",
+        "order" : 1
```

Comment:
You should either change this to 3 (to make it show below read_timeout, or place it before the other parameters here.

You also need to run `make xml-upload` and attach the updated metadata to this PR, so it doesnt fail CI.

Reply:
Thanks, done.